express-101/express101.js

```diff
@@ -1,8 +1,10 @@
+const express = require('express')
+const app = express()
+const port = 3000
 
-function express101(...args) {
-  return args;
-}
+app.get('/', (req, res) => res.send('Hey, I am server response'))
 
-export {
-  express101,
-};
+app.get('/movie/:title', (req, res) => res.send(`My favorite movie is ${req.params['title']}`))
```

Comment:
Won't the use of destructuring here make further modifications easy?
Like so: const { title } = req.params;
You won't have to rewrite req.params.title everytime.

Reply:
Yup, that's true. But I was trying to avoid adding so many variables since the query parameter clarifies that. Would you still suggest a change is warranted?